index.html

```diff
@@ -31,13 +32,210 @@ <h2 id="job-title">I am a Software Developer</h2>
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 55 (1).png" alt=""><p id="framework-text">Language </p></div>
+                            <li id="individual-languages">JavaSript</li>
+                            <li id="individual-languages">Ruby</li>
+                            <li id="individual-languages">Html</li>
+                            <li id="individual-languages">Css</li>
+                    </div>
+                    <div class="frameworks-div">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 56 (1).png" alt=""><p id="framework-text">Frameworks </p></div>
+                            <div class="mini-language-div">
+                                <li id="individual-frameworks">Bootstrap</li>
+                                <li id="individual-frameworks">Ruby on Rails</li>
+                                <li id="individual-frameworks">RSPec</li>
+                                <li  id="individual-frameworks">CapyBara</li>
+                                <li id="individual-frameworks">Selenium</li>
+                            </div>
+                            
+                        </ul>
+                    </div>
+                    <div class="skills-div">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 57 (1).png" alt=""><p id="framework-text">Skills </p></div>
+                            <li id="individual-skills">Codekit</li>
+                            <li id="individual-skills">GitHub</li>
+                            <li id="individual-skills">Codepen</li>
+                            <li id="individual-skills">Gitlab</li>
+                            <li id="individual-skills">Terminal</li>
+                    </div>
+                </div>
+            </section>
+            <section class="footnote">
+                <div id="brief-note-div">
+                    <h3 id="brief-note">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                </div>
+                <div class="get-in-touch">
+                    <input class="get-in-touch-individuals" type="text" placeholder="Full name">
+                    <input type="text" placeholder="Email address">
+                    <input type="text" placeholder="Enter text here">
+                    <a href=""><button id="get-in-touch-button">Get in touch</button></a>
+                </div>
+                <p id="line1">line</p>
+                <div class="footnote-socials">
+                    <a href=""><img src="images/github.png" alt=""></a>
+                    <a href=""><img src="images/linkedin.png" alt=""></a>
+                    <a href=""><img src="images/hand.png" alt=""></a>
+                    <a href=""><img src="images/twitter.png" alt=""></a>
+                    <a href=""><img src="images/Vector.png" alt=""></a>
```

Comment:
Kindly, this suggestion that I did the last revision did not do it, I recommend doing it. Put "#" in the "href". use this suggestion in all "A" tags

Reply:
Kindly, this suggestion that I did the last revision did not do it, I recommend doing it. put some "alt" text. use this suggestion in all "img" tags